### ISSUE TITLE
Implement gear AutoInstruction

### ISSUE THREAD

**Issue by randomtestfive** [RAR1741/RA17_RobotCode#43]
It's just a comment right now

**Comment by dracco1993** [comment]
Yeah, this should really be the next thing we work on, as it's the most important thing we can do in auto, outside of actually driving.  😂

**Comment by randomtestfive** [comment]
Added by bd083560b9913f1542b04eb4c7a4f977f3d4d808
